src/app.jsx

```diff
@@ -406,6 +407,9 @@ class Application extends React.Component {
                     }
                 })
                 .fail(e => console.log("Could not read $XDG_RUNTIME_DIR: ", e.message));
+        cockpit.spawn("selinuxenabled")
```

Comment:
I checked that `selinuxenabled` works as non-root. But I think this should use `{ error: "ignore" }` to not spam the journal with "no such command" etc. errors.

Reply:
updated